package.json

```diff
@@ -719,7 +737,13 @@
           "icon": "images/activity-bar-icon.svg"
         }
       ]
-    }
+    },
+    "jsonValidation": [
+      {
+        "fileMatch": "playground.json",
+        "url": "https://gist.githubusercontent.com/legomushroom/e6ef081dc3f466b6228acac5ee9b620d/raw/gistpad-schema.json"
```

Comment:
Can't this refer to a file in the VSIX? It would be ideal if we could ship this with the extension itself.

Reply:
Ya, but we cannot update the schema for the older clients without deployment then.

Reply:
Oh also the serverless scenarios. We need to host the file somewhere for that.